### ISSUE TITLE
sudo: unable to resolve host facility: Name or service not known

### ISSUE THREAD

**Issue by 0xbkt** [tailscale/tailscale#968]
**Describe the bug**
Running `sudo` prints out this warning message in the first line. 

**To Reproduce**
Run `sudo anything`.

**Expected behavior**
No warning.

**Screenshots**
![Screenshot from 2020-11-26 19-54-02](https://user-images.githubusercontent.com/41230766/100376808-32d7f580-3021-11eb-8a15-6d66b24a6566.png)

**Version information:**
```
                             owner@facility
                             OS: Pop 20.04 focal
                             Kernel: x86_64 Linux 5.8.0-7630-generic
         #####               Uptime: 1d 8h 58m
        #######              Packages: Unknown
        ##O#O##              Shell: bash 5.0.17
        #######              Resolution: 1920x1080
      ###########            DE: GNOME 3.38.1
     #############           WM: Mutter
    ###############          WM Theme: Pop
    ################         GTK Theme: Pop-dark [GTK2/3]
   #################         Icon Theme: Pop
 #####################       Font: Fira Sans Semi-Light 10
 #####################       Disk: 15G / 325G (5%)
   #################         CPU: Intel Core i3-6100 @ 4x 3.7GHz [40.0°C]
                             GPU: GeForce GTX 950
                             RAM: 6309MiB / 7918MiB
```
```bash
owner@facility:~$ tailscale version
1.2.8
  tailscale commit: cde3a23b6649ed448a9a94bd776043fb09e966d5
  other commit: 1f7ecb611d82fb6f95f6a65fd61fc2265f98dc37
  go version: go1.15.4-tsf9db43b
```

**Additional context:**
Magic DNS is on.

**Comment by danderson** [comment]
Catching up on DNS bugs: is this still happening for you on Tailscale 1.6? If so, can I have the output of the following commands (all run as root) ?

```
cat /etc/resolv.conf
find /run/resolvconf
resolvectl
nmcli
```

**Comment by 0xbkt** [comment]
@danderson it must be fixed as i'm not having the same behavior with the latest version. thanks.

**Comment by 0xbkt** [comment]
oh, wait. it was with magic dns on. yes, i see the same problem with the latest.

**Comment by 0xbkt** [comment]
Just did a fresh install of PopOS 20.04 and now it's gone. I'm not really sure what has changed since, because it's a practice of mine to not mess with DNS/network configuration unless I'm desperate. I suppose something might be broken with the version I was using back then. Thank you for building Tailscale.

**Comment by danderson** [comment]
Wow, hat trick of DNS services on that machine! Congrats :)

So, the good news is, I think you were a victim of one of the various ways in which we detected DNS configs wrong on linux, which are all fixed on 1.8 (coming in a week or two). If you want to give that a test drive, you could install the unstable linux packages from pkgs.tailscale.com. We haven't applied the spit and polish yet, so things might still break, but on your system I believe the unstable build would end up programming DNS through NetworkManager, which will then bubble down through the layers and plug in gracefully with everything else.

**Comment by danderson** [comment]
Going to close this bug on the belief that 1.8 fixes it, and you also got things happy with a reinstall, but please do reopen if things go wrong again and I need to investigate!
